Cargo.toml

```diff
@@ -25,7 +25,7 @@ duration-str = { version = "0.11.2", default-features = false }
 expect-test = "1.5.0"
 fake = "2.9.2"
 fs-err = "2.11.0"
-git-cliff-core = { version = "2.4.0", default-features = false }
+git-cliff-core = { version = "2.4.0", default-features = false, features = ["github", "gitea"] }
```

Comment:
it feels bad that we want to just add some values and our cargo.lock exploded. Maybe we can edit git-cliff to add arbitrary fields to the commits, so that we don't need to enable these features.

Reply:
opened https://github.com/orhun/git-cliff/issues/803